cutplace/applications.py

```diff
@@ -165,6 +172,20 @@ def process(argv=None):
     if cutplace_app.is_gui:
         # TODO #77: Open graphical user interface.
         raise NotImplementedError
+    elif cutplace_app.is_create_sql:
```

Comment:
Move the inner part of this `if` section to a separate function in module `sql` to keep the code clean, e.g. `sql.write_create(cid_path)`